data_gatherer.py

```diff
@@ -64,17 +71,16 @@ def main(argv):
         control_loop_stream, release_sensor_stream,
         pipeline_finish_notify_stream)
 
-    # Add sensors.
-    (center_camera_stream, notify_rgb_stream,
-     rgb_camera_setup) = pylot.operator_creator.add_rgb_camera(
-         transform, vehicle_id_stream, release_sensor_stream)
-    (depth_camera_stream, _,
-     depth_camera_setup) = pylot.operator_creator.add_depth_camera(
-         transform, vehicle_id_stream, release_sensor_stream)
-    (segmented_stream, _,
-     _) = pylot.operator_creator.add_segmented_camera(transform,
-                                                      vehicle_id_stream,
-                                                      release_sensor_stream)
+    if not FLAGS.perfect_perception:
```

Comment:
Can we rename this to `skip_cameras` for now? and before we push it in, we can define a function that ensures consistency of this flag with the other flags.